taichi/ui/backends/vulkan/app_context.h

```diff
@@ -37,13 +37,36 @@ class TI_DLL_EXPORT AppContext {
 
   AppConfig config;
 
+  taichi::lang::Pipeline *get_raster_pipeline(
```

Comment:
style nit: pack these params into a dedicated params struct, so that we don't run into risks of confusing param order && avoids changing the interface when we need to add more parameters